recipe/meta.yaml

```diff
@@ -9,8 +9,8 @@ package:
   version: "{{ version }}"
 
 source:
-  url: http://www.netlib.org/lapack/lapack-{{ version }}.tar.gz
-  sha256: deb22cc4a6120bff72621155a9917f485f96ef8319ac074a7afbc68aab88bcf6
```

Comment:
The url for the sources [changed](http://www.netlib.org/lapack/#_lapack_version_3_9_0_2) between 3.8.0 and 3.9.0